zerver/middleware.py

```diff
@@ -498,7 +485,9 @@ def csrf_failure(request: HttpRequest, reason: str = "") -> HttpResponse:
 
 
 class LocaleMiddleware(DjangoLocaleMiddleware):
-    def process_response(self, request: HttpRequest, response: HttpResponse) -> HttpResponse:
+    def process_response(
+        self, request: HttpRequest, response: HttpResponseBase
+    ) -> HttpResponseBase:
```

Comment:
What's the different between `HttpResponse` and `HttpResponseBase`?  I'm not sure if this is correct.

Reply:
For some reason, the supertype of this middleware uses `HttpResponseBase` in its function signature for `process_response`, which makes our `process_response` incompatible with it. `HttpResponseBase` does not handle the content of a response (but it does deal with cookies and headers), which is actually sufficient in our use case for the locale middleware.

Reply:
I think this is correct; it could be a `StreamingHttpResponse`